src/SprykerEco/Zed/CrefoPay/Persistence/CrefoPayRepository.php

```diff
@@ -94,23 +94,43 @@ public function findPaymentCrefoPayByIdSalesOrderItem(int $idSalesOrderItem): Pa
 
     /**
      * @param string $crefoPayOrderId
-     * @param string|null $captureId
      *
      * @return \Generated\Shared\Transfer\PaymentCrefoPayOrderItemCollectionTransfer
      */
-    public function findPaymentCrefoPayOrderItemsByCrefoPayOrderIdAndCaptureId(
-        string $crefoPayOrderId,
-        ?string $captureId = null
-    ): PaymentCrefoPayOrderItemCollectionTransfer {
+    public function findPaymentCrefoPayOrderItemsByCrefoPayOrderId(string $crefoPayOrderId): PaymentCrefoPayOrderItemCollectionTransfer
+    {
         $query = $this->getPaymentCrefoPayOrderItemQuery();
         $query->useSpyPaymentCrefoPayQuery()
                 ->filterByCrefoPayOrderId($crefoPayOrderId)
             ->endUse();
 
-        if ($captureId !== null) {
-            $query->filterByCaptureId($captureId);
+        $paymentCrefoPayOrderItemEntities = $query->find();
```

Comment:
IMO you can omit $query variable now and fetch data directly to $paymentCrefoPayOrderItemEntities variable